docs: update addresses

0xMonoAx opened the pull request:
Updated all documentation under `addresses/`, including:
- `csa-current-efforts`
- `csa-user-stories`
- `interoperable-address-explainer`
- `interoperable-addresses-caip10-did-compatibility`
- `interoperable-onchain-chain-names`

0xMonoAx committed:
docs: update-csa-current-efforts

0xMonoAx committed:
docs: update-csa-user-stories

0xMonoAx committed:
docs: update-interoperable-address-explainer

0xMonoAx committed:
docs: update-interoperable-addresses-caip10-did-compatibility

0xMonoAx committed:
docs: update-interoperable-onchain-chain-names

0xrcinus commented on docs/addresses/csa-current-efforts.md:
```diff
@@ -1,44 +1,69 @@
+
+| **Feature** | ERC-3770 | CAIP-10 | CAIP-50 | ERC-7828 | ENSIP-9/ENSIP-11 | CAIP-2 | ERC-7930 |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| **Scope** | Primarily a UI/UX layer standard for human-readable prefixes | A universal, machine-readable account identifier format for all blockchains | Compact, binary account ID format for all blockchains | On-chain naming integration with ENS for chain and address names, EVM-focused | ENS resolver-level standards for storing/retrieving multichain addresses | Universal blockchain identifier ( ...
```
Is 7828 EVM focused? I think the intention is to also support non-EVM chains

0xMonoAx replied:
You're right, it also supports non-EVM chains

0xrcinus commented on docs/addresses/csa-current-efforts.md:
```diff
@@ -1,44 +1,69 @@
 # Chain-specific addresses
 
-A core challenge in Ethereum's multichain landscape is disambiguating addresses across L2s, L3s, Ln, and other EVM ecosystems. In this context, an address by itself does not provide enough information to determine which chain it belongs to, and in the case of smart contracts, its code content may even differ. For this reason, the introduction of chain identifiers and human-readable labels aims to ...
+A core challenge in Ethereum's multichain landscape is disambiguating addresses across L2s, L3s, Lns, and other EVM ecosystems. In this context, an address by itself does not provide enough information to determine which chain it belongs to.
```
I don't know if "belongs to" is the right framing, maybe: "As a given Ethereum address can exist on any EVM chain, applications and users need a way to also specify a target chain, to avoid errors and confusion."

0xrcinus reviewed (approved):
Small comments, but looks good!

0xMonoAx committed:
Update csa-current-efforts.md

0xMonoAx committed:
Update interoperable-address-explainer.md

The `interoperable-addresses-caip10-did-compatibility` document was merged with `interoperable-address-explainer` to simplify and consolidate the documentation into a single, coherent reference. As a result of this unification, `interoperable-addresses-caip10-did-compatibility` will be removed, as its content has been fully integrated into the main document.

0xMonoAx committed:
Delete docs/addresses/interoperable-addresses-caip10-did-compatibility.md

Joxess commented on docs/addresses/csa-current-efforts.md:
```diff
@@ -1,44 +1,71 @@
+### ENSIP-11: EVM-compatible Chain Address Resolution
+[ENSIP-11](https://github.com/ensdomains/ensips/blob/master/ensips/11.md) extends ENSIP-9 by introducing a dedicated range of coin types for EVM chains. This range prevents collisions with existing [SLIP-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) coin types.
 
 ## Comparison
```
Should we add CAIP-350 here as well?

Joxess replied:
Maybe is not worth it, but perhaps and "*" note below the table on why is not there could also help

0xMonoAx replied:
I have added the description of the CAIP-350 to the box to the table

Joxess commented on docs/addresses/interoperable-address-explainer.md:
```diff
@@ -1,54 +1,80 @@
+# Interoperable Addresses Explained
 
-*Formal format specifications are written [here](/specs/addresses/cross-chain-interoperable-addresses-spec.md)*.
+*The formal specification can be found in [EIP-7930](https://eips.ethereum.org/EIPS/eip-7930).*
```
```suggestion
*The formal specification can be found in [ERC-7930](https://eips.ethereum.org/EIPS/eip-7930).*
```

Joxess commented on docs/addresses/interoperable-address-explainer.md:
```diff
@@ -1,54 +1,80 @@
-Introducing: Interoperable Addresses
-====
+# Interoperable Addresses Explained
```
```suggestion
# ERC-7930: Interoperable Addresses, Explained
```

or something like that

Joxess commented on docs/addresses/interoperable-address-explainer.md:
````diff
@@ -1,54 +1,80 @@
+└─────────┴───────────┴──────────────────────┴────────────────┴───────────────┴─────────┘
+```
+- **Version**: A 2-byte identifier that allows for future upgrades.
+- **ChainType**: A 2-byte value defining the blockchain family (e.g., `eip155` for EVM).
````
Maybe we should make this more clear saying that might corresponds to a CAIP-2 namespace (if CAIP-350 is used)

0xMonoAx replied:
I changed it to the following description:

> A 2-byte value as defined in CAIP-350, corresponding to a CAIP-2 namespace, which allows users to know how to interpret & display the other two fields.

Joxess reviewed (commented):
Looking very good, just tiny comments !

0xMonoAx committed:
Update docs/addresses/interoperable-address-explainer.md

Co-authored-by: Joxes <91908708+Joxess@users.noreply.github.com>

0xMonoAx committed:
Update csa-current-efforts

0xMonoAx committed:
Update docs/addresses/interoperable-address-explainer.md

Co-authored-by: Joxes <91908708+Joxess@users.noreply.github.com>

0xMonoAx committed:
Update interoperable-address-explainer.md

0xMonoAx commented:
I have updated the documentation according to Joxe's comments.
